app/editor/Dockerfile

```diff
@@ -5,6 +5,7 @@ COPY .yarn ./.yarn
 COPY package*.json yarn.lock .yarnrc.yml ./
 
 RUN yarn set version berry
+RUN npx browserslist@latest --update-db
```

Comment:
When we built the image in Openshift it kept giving a warning stating we should do this.